ios/Capacitor/Capacitor/CAPLog.swift

```diff
@@ -1,19 +1,11 @@
 public class CAPLog {
-
-    public static let config = CAPConfig()
+    public static var enableLogging: Bool = true
 
     public static func print(_ items: Any..., separator: String = " ", terminator: String = "\n") {
-        if !self.hideLogs() {
+        if enableLogging {
             for (itemIndex, item) in items.enumerated() {
                 Swift.print(item, terminator: itemIndex == items.count - 1 ? terminator : separator)
             }
         }
     }
-
-    public static func hideLogs() -> Bool {
```

Comment:
Not sure why this was public to begin with, but just noting that it is something being removed from the public API.

Reply:
Yeah, its replacement property is public as well but we should probably note that.